src/main/kotlin/nl/info/zac/admin/ZaaktypeCmmnConfigurationBeheerService.kt

```diff
@@ -205,6 +207,34 @@ class ZaaktypeCmmnConfigurationBeheerService @Inject constructor(
         }
     }
 
+    fun checkIfProductaanvraagtypeIsNotAlreadyInUse(
+        productaanvraagtype: String,
+        zaaktypeOmschrijving: String
+    ) {
+        val activeZaaktypeCmmnConfigurationForProductaanvraagtype =
```

Comment:
Just a minor question about naming: `activeZaaktypeCmmnConfiguration` sounds like (CMMN) ZAPs or is this something different? Why don't we use the term `zaakafhandelparameters` here instead? Or have we decided to move away from the term `zaakafhandelparameters` altogether? I don't think so right? At least not in the UI.